libdnf5/rpm/package.cpp

```diff
@@ -329,7 +329,7 @@ std::vector<std::string> Package::get_remote_locations(const std::set<std::strin
     auto repo_mirrors = get_repo()->get_mirrors();
     for (const auto & mirror : repo_mirrors) {
         for (const auto & protocol : protocols) {
-            if (utils::string::starts_with(mirror, protocol)) {
+            if (utils::string::starts_with(mirror, protocol + ":")) {
```

Comment:
Here is missing the ``Ensure the urls...`` comment. Either copy it there, or remove it from the other occurance at line 344.

Reply:
Opted to delete the comment. It self explanatory